profiling-ffi/src/exporter.rs

```diff
@@ -169,6 +170,28 @@ unsafe fn into_vec_files<'a>(slice: Slice<'a, File>) -> Vec<exporter::File<'a>>
         .collect()
 }
 
+#[repr(C)]
+pub enum ExporterRequestBuildResult {
+    Ok(Box<Request>),
+    Err(ddcommon_ffi::Vec<u8>),
+}
+
+#[cfg(test)]
+impl From<ExporterRequestBuildResult> for Result<Box<Request>, String> {
+    fn from(result: ExporterRequestBuildResult) -> Self {
+        match result {
+            ExporterRequestBuildResult::Ok(ok) => Ok(ok),
+            ExporterRequestBuildResult::Err(err) => {
+                // Safety: not generally safe but this is for test code.
+                Err(unsafe { String::from_utf8_unchecked(err.into()) })
+            }
+        }
+    }
+}
+
+#[no_mangle]
+pub extern "C" fn ddog_prof_ExporterRequestBuildResult_drop(_result: ExporterRequestBuildResult) {}
```

Comment:
Since we started from `ddog_prof_Exporter_Request_build` I think this should be `ddog_prof_Exporter_Request_BuildResult_drop`

(Similar to how `ddog_prof_Exporter_send` => `ddog_prof_Exporter_SendResult_drop`)

```suggestion
pub extern "C" fn ddog_prof_Exporter_Request_BuildResult_drop(_result: ExporterRequestBuildResult) {}
```